rtengine/iimage.h

```diff
@@ -98,6 +98,10 @@ class ImageDatas : virtual public ImageDimensions
         return "unknown";
     }
 
+    virtual bool isBW () const
```

Comment:
I know it is currently done everywhere, but I think inlining the virtuals implies vague linkage? Shouldn't it just be moved into `iimage.cc`?